livekit-android-camerax/src/main/java/livekit/org/webrtc/CameraXSession.kt

```diff
@@ -147,7 +147,7 @@ internal constructor(
 
                 surface = Surface(surfaceTextureHelper.surfaceTexture)
                 surfaceProvider = SurfaceProvider { request ->
-                    request.provideSurface(surface!!, helperExecutor) { }
```

Comment:
Surface is only nulled out when the camera is stopped, so we'll need to call `request.willNotComplete()` if we're not providing it any more.

https://developer.android.com/reference/androidx/camera/core/Preview.SurfaceProvider#onSurfaceRequested(androidx.camera.core.SurfaceRequest)